var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -146,7 +148,9 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     variant("monolithic", default=False, description="Static monolithic build")
     variant("numa", default=False, description="Build with NUMA support")
```

Comment:
```suggestion
    variant("numa", default=True, description="Build with NUMA support")
```

Reply:
Same with `build:mkl`, I think the `build:numa` section is only activated conditionally